common/aggregate_functions/src/aggregate_function_test.rs

```diff
@@ -159,3 +159,143 @@ fn test_aggregate_function() -> Result<()> {
+    struct Test {
+        name: &'static str,
+        eval_nums: usize,
+        args: Vec<DataField>,
+        display: &'static str,
+        columns: Vec<DataColumnarValue>,
+        expect: DataValue,
+        error: &'static str,
+        func_name: &'static str,
+    }
+
+    let columns: Vec<DataColumnarValue> = vec![
+        Arc::new(Int64Array::builder(0).finish()).into(),
+        Arc::new(Int64Array::builder(0).finish()).into(),
+    ];
+
+    let args = vec![
+        DataField::new("a", DataType::Int64, true),
+        DataField::new("b", DataType::Int64, true),
+    ];
+
+    let tests = vec![
+        Test {
+            name: "count-passed",
+            eval_nums: 1,
+            args: vec![args[0].clone()],
+            display: "count",
+            func_name: "count",
+            columns: vec![columns[0].clone()],
+            expect: DataValue::UInt64(Some(0)),
+            error: "",
+        },
+        Test {
+            name: "max-passed",
+            eval_nums: 2,
+            args: vec![args[0].clone()],
+            display: "max",
+            func_name: "max",
+            columns: vec![columns[0].clone()],
+            expect: DataValue::Null,
+            error: "",
+        },
+        Test {
+            name: "min-passed",
+            eval_nums: 2,
+            args: vec![args[0].clone()],
+            display: "min",
+            func_name: "min",
+            columns: vec![columns[0].clone()],
+            expect: DataValue::Null,
+            error: "",
+        },
+        Test {
+            name: "avg-passed",
+            eval_nums: 1,
+            args: vec![args[0].clone()],
+            display: "avg",
+            func_name: "avg",
+            columns: vec![columns[0].clone()],
+            expect: DataValue::Null,
+            error: "",
+        },
+        Test {
+            name: "sum-passed",
+            eval_nums: 1,
+            args: vec![args[0].clone()],
+            display: "sum",
+            func_name: "sum",
+            columns: vec![columns[0].clone()],
+            expect: DataValue::Null,
+            error: "",
+        },
+        Test {
+            name: "argMax-passed",
+            eval_nums: 1,
+            args: args.clone(),
+            display: "argmax",
+            func_name: "argmax",
+            columns: columns.clone(),
+            expect: DataValue::Null,
+            error: "",
+        },
+        Test {
+            name: "argMin-passed",
+            eval_nums: 1,
+            args: args.clone(),
+            display: "argmin",
+            func_name: "argmin",
+            columns: columns.clone(),
+            expect: DataValue::Null,
+            error: "",
+        },
+        Test {
+            name: "uniq-passed",
+            eval_nums: 1,
+            args: vec![args[0].clone()],
+            display: "uniq",
+            func_name: "uniq",
+            columns: vec![columns[0].clone()],
+            expect: DataValue::UInt64(Some(0)),
```

Comment:
maybe uniq function should return empty rows?

Reply:
The uniq function seems to behave the same as count distinct, so I return 0